pkg/moq/moq_test.go

```diff
@@ -664,6 +664,22 @@ func TestImportedPackageWithSameName(t *testing.T) {
 	}
 }
 
+func TestImportedPackageWithTypeAlias(t *testing.T) {
```

Comment:
Can you use the golden file test similar to https://github.com/matryer/moq/blob/v0.4.0/pkg/moq/moq_test.go#L409-L414?

Reply:
Ah yeah I hadn't worked out how those golden files worked, that looks much better! Updated, thanks!